src/main/java/com/github/cross_language_cpp/djinni_intellij_plugin/DjinniBlock.java

```diff
@@ -48,7 +48,7 @@ protected DjinniBlock(@NotNull ASTNode node,
     this.spaceIndent = spaceIndent;
   }
 
-  static List<IElementType> memberTypes = Arrays.asList(DjinniTypes.ENUM_MEMBER, DjinniTypes.RECORD_MEMBER, DjinniTypes.INTERFACE_MEMBER, DjinniTypes.CONST_RECORD_MEMBER_ELEMENT);
+  static List<IElementType> memberTypes = Arrays.asList(DjinniTypes.ENUM_MEMBER, DjinniTypes.FLAGS_MEMBER, DjinniTypes.RECORD_MEMBER, DjinniTypes.INTERFACE_MEMBER, DjinniTypes.CONST_RECORD_MEMBER_ELEMENT);
```

Comment:
I had to do many adjustments like this one. I did not attemt to understand what is going on. All I did was searching the codebase for `enum` appearances and copy and rename them to `flags`. Structurally `enum` and `flags` are nearly identically, thats why this strategy seems to work.